.vscode/launch.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
`.gitignore` this file 😄

Reply:
So the current .gitignore file is explicitly allowing this file:

```
### VisualStudioCode ###
.vscode/*
!.vscode/settings.json
!.vscode/tasks.json
!.vscode/launch.json
!.vscode/extensions.json
.history
```

Reply:
I'd rather keep editor-specific config out of the repo if possible. This seems like it would be totally fine untracked.